notebooks/404-style-transfer-webcam/404-style-transfer.ipynb

```diff
@@ -0,0 +1,482 @@
+    "\n",
+    "<img align='center' src=\"https://user-images.githubusercontent.com/109281183/204767232-95b8f0a8-010e-4f6d-981e-104200652098.jpg\" alt=\"drawing\" width=\"1000\"/>\n",
+    "\n",
+    "This notebook uses five pre-trained models, for the following styles: Mosaic, Rain Princess, Candy, Udnie and Pointilism. The models are from the [ONNX Model Repository](https://github.com/onnx/models) and are based on the research paper [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155) along with [Instance Normalization](https://arxiv.org/abs/1607.08022). Final part of this notebook shows live inference results from a webcam. Additionally, you can also upload a video file.\n",
+    "\n",
+    "> **NOTE**: If you have a webcam available on your computer, you can see the result in the live video streaming in the notebook. Otherwise, if you run the notebook on a server, the webcam won't work, and you can make inference on a video file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import collections\n",
+    "import sys\n",
+    "import time\n",
+    "\n",
+    "import cv2\n",
+    "import numpy as np\n",
+    "from pathlib import Path\n",
+    "from IPython.display import Markdown as md\n",
+    "from IPython import display\n",
+    "from ipywidgets import interactive, ToggleButtons\n",
+    "from openvino.runtime import Core\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Select one of the styles: Mosaic, Rain Princess, Candy, Udnie, and Pointilism from the button below to do the style transfer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Option to select different styles    \n",
+    "styleButtons = ToggleButtons(\n",
+    "    options=['MOSAIC', 'RAIN-PRINCESS', 'CANDY', 'UDNIE', 'POINTILISM'],\n",
+    "    description=\"Click one of the styles you want to use for the style transfer\",\n",
+    "    disabled=False,\n",
+    "    style={'description_width': '300px'})\n",
+    "   \n",
+    "interactive(lambda option: print(option), option=styleButtons)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Model\n",
+    "\n",
+    "### Download the Model\n",
+    "\n",
+    "Model for the selected style will be downloaded to `model_path` if they have not been downloaded before. The selected model comes from the ONNX Model Zoo, which means it could be used with OpenVINO directly but it is recommended to be converted into Intermediate Representation (OpenVINO IR)."
```

Comment:
Why is recommended to convert to IR? Is this not working well with ONNX? We need to be careful with these comments.  I think we could change that to "..., but in this example, we like to show you how to use the Model Optimizer to convert ONNX to IR.... or something like that.  Or avoid that explanation.

Reply:
@ryanloney This notebook is ready for your review.

Reply:
updated the content based on your suggestion.
"Model for the selected style will be downloaded to model_path if they have not been downloaded before. The selected model comes from the ONNX Model Zoo, which means it could be used with OpenVINO directly but in this notebook, we like to show you how to use the Model Optimizer to convert ONNX to Intermediate Representation (OpenVINO IR)."

Reply:
@paularamo it will work with ONNX, but slower. So it's recommended to use IR as often as possible.